### PR TITLE
Update bio, name and username

### PR DESCRIPTION
**Pull request by valeyko**

### CODE REVIEW

File: backend-service/src/main/java/edu/clubhouseapi/controller/UserController.java [hunk 1]
```diff
@@ -80,4 +69,19 @@
...
         return clubHouseUserApiService.getNotifications(pageSize, page);
     }
+
+    @PostMapping("/api/update_bio")
+    public Mono<EmptyResponse> updateBio(@RequestBody @Valid UpdateBioRequest updateBioRequest) {
+        return clubHouseUserApiService.updateBio(updateBioRequest);
+    }
+
+    @PostMapping("/api/update_name")
+    public Mono<EmptyResponse> updateBio(@RequestBody @Valid UpdateNameRequest updateNameRequest) {
```
Comment (seovchinnikov) [inline backend-service/src/main/java/edu/clubhouseapi/controller/UserController.java]: lets fix method name

File: backend-service/src/main/java/edu/clubhouseapi/controller/UserController.java [hunk 2]
```diff
@@ -80,4 +69,19 @@
...
+        return clubHouseUserApiService.updateName(updateNameRequest);
+    }
+
+    @PostMapping("/api/update_username")
+    public Mono<EmptyResponse> updateBio(@RequestBody @Valid UpdateUsernameRequest updateUsernameRequest) {
```
Comment (seovchinnikov) [inline backend-service/src/main/java/edu/clubhouseapi/controller/UserController.java]: lets fix method name

File: frontend/src/components/MyProfile.vue [hunk 3]
```diff
@@ -156,6 +216,10 @@
         private message: string = "";
         private userData!: UserProfileInfoResponse;
         private avatar: string = '/img/no-avatar.png';
+        private showEditName: boolean = false;
+        private showEditUsername: boolean = false;
+        private name: string = "";
+        private username: string = "";
```
Comment (seovchinnikov) [inline frontend/src/components/MyProfile.vue]: maybe rename these one (name, username)  to something like editableName, editableUsername?

File: backend-service/src/main/java/edu/clubhouseapi/controller/UserController.java (hunk 1 above)
Comment (valeyko) [reply backend-service/src/main/java/edu/clubhouseapi/controller/UserController.java]: done

**Review by seovchinnikov — APPROVED** [review]

**Pull request merged**